api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -8081,3 +8082,29 @@ message StableUNIXUser {
   string username = 1;
   int32 uid = 2;
 }
+
+// AWSICResourceSync is emitted when the AWS Identity Center integration
+// imports resources from Identity Center and reconciles them to Teleport.
+message AWSICResourceSync {
+  // Metadata is the event metadata.
+  Metadata metadata = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
+  // Total imported AWS accounts.
+  int32 total_account = 2 [(gogoproto.jsontag) = "total_account,omitempty"];
```

Comment:
```suggestion
  int32 total_accounts = 2 [(gogoproto.jsontag) = "total_account,omitempty"];
```

Can we use plural ? that will make the fields naming consitent with other UsageEvents like: 
https://github.com/gravitational/teleport/blob/7b6f291fb348850f7e621a5e102fa0094f32bdd1/api/proto/teleport/usageevents/v1/usageevents.proto#L753

Reply:
https://github.com/gravitational/teleport/pull/52059/commits/980ba2a98b9dc19b87d962caa6490ee39101b768